### PR TITLE
feat: implement LiteLLM for standardized LLM API calls for Embeddings

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
fix https://github.com/aietal/isaac/issues/28

### CODE REVIEW

**Comment by rishi-raj-jain** [comment]
Would love to get this in and get onto the next one!

**Comment by rishi-raj-jain** [comment]
Would love to get this in and get onto the next one!

File: web/utils/create_embedding.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export async function createEmbedding(body) {
+    const embedding = await fetch('http://0.0.0.0:5001/api/embedding', {
```
[the remark this answers is not in the document]
Comment (rishi-raj-jain) [reply web/utils/create_embedding.ts:2]: The same as done in the last PR

**Review by eimenhmdt — CHANGES_REQUESTED** [review]
Code lgtm. Would be great, however, if you could replace the hardcoded host and api key in create_embedding.ts with the following env vars:

api host: process.env.NEXT_PUBLIC_API_HOST
api key: process.env.NEXT_PUBLIC_API_ROUTE_SECRET 

Thanks!

**Review by eimenhmdt — APPROVED** [review]
Thanks for the modifications. Approved!

**Pull request merged**